### ISSUE TITLE
Use ModelID and SetModelID to appease golint

### ISSUE THREAD

**Issue by penguinpowernz** [albrow/zoom#27]
Currently the interface is:

```go
type Model interface {
  ModelId() string
  SetModelId(string)
}
```

But that causes `golint` to whinge.  The correct way according to go is:

```go
type Model interface {
  ModelID() string
  SetModelID(string)
}
```

Will obviously have to manage people already using this, maybe will need to bump to v2?

**Comment by albrow** [comment]
@penguinpowernz I fixed the issue and released [Version 0.19.0](https://github.com/albrow/zoom/releases/tag/v0.19.0). Because [semantic versioning](https://semver.org/) allows for breaking changes before version 1.0, I elected not to bump the major version at this time. Zoom is still on version 0.x and the [README](https://github.com/albrow/zoom/blob/master/README.md#development-status) is pretty clear about this and what it means.

**Comment by penguinpowernz** [comment]
Awesome dude! :+1:
